en/installation-guide/database-initialization.rst

```diff
@@ -75,9 +73,11 @@ Restore the latest data model that also includes demo data:
 
 .. figure:: images/pgadmin_qgep_refresh.jpg
 
-* Update privileges for the database
+Since database release 1.0 you will get 3 new schemas (qgep_od, qgep_sys, qgep_vl)
```

Comment:
I think there is no reason to mention that it's since release 1.0. For anyone new, they won't know the old schema, and everyone who was there before should already know.